wasm/jit/jit_value_location_amd64.go

```diff
@@ -155,6 +156,9 @@ func (s *valueLocationStack) push(loc *valueLocation) {
 	} else {
 		s.stack[s.sp] = loc
 	}
+	if s.sp > s.maxHeight {
```

Comment:
max is a word usually implying a boundary, but this feels like it will change any time `sp` changes. for that reason, I feel it is useful to add a comment like `// allow the go allocated stack to grow to accommodate the stack pointer`

even if we comment, though I'd still feel like it isn't so much a `max` as a length thing, as it is adjusted whenever it needs to be... wdyt?

Reply:
it's not about the length of slice here, but about the stack pointer. So I will rename `maxHeight` to `maxStackPointer`  to clarify!

Reply:
💯

Reply:
https://github.com/tetratelabs/wazero/pull/68/commits/31611a5160d80b58483a08618af219830922b84c done! Thanks for the comment!